src/Magnum/LibOvrIntegration/HmdEnum.h

```diff
@@ -0,0 +1,105 @@
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+/** @file
+ * @brief Wrapped libOVR enums HmdType, HmdCapability, HmdTrackingCapability and HmdStatusFlag.
+ *
+ * @author Jonathan Hale (Squareys)
+ */
+
+#include <memory>
+#include <Magnum/Texture.h>
+#include <Magnum/Magnum.h>
+
+#include <OVR/OVR_CAPI.h>
+#include "visibility.h"
+
+namespace Magnum { namespace LibOvrIntegration {
+
+enum class HmdType: UnsignedByte {
+    None = ovrHmd_None,
+    DK1 = ovrHmd_DK1,
+    DKHD = ovrHmd_DKHD,
+    DK2 = ovrHmd_DK2,
+    BlackStar = ovrHmd_BlackStar,
+    CB = ovrHmd_CB,
+    Other = ovrHmd_Other
+};
+
+enum class HmdCapability: UnsignedInt {
+    /**
+     *  @brief Toggles low persistence mode on or off.
+     *  @details This setting reduces eye-tracking based motion blur. Eye-tracking based motion blur is caused by the viewer's focal point
+     *  moving more pixels than have refreshed in the same period of time.\n
+     *  The disadvantage of this setting is that this reduces the average brightness of the display and causes some users to perceive flicker.\n
+     *  <I>There is no performance cost for this option. Oculus recommends exposing it to the user as an optional setting.</I> */
+    LowPersistence = ovrHmdCap_LowPersistence,
+
+    /** Adjusts prediction dynamically based on internally measured latency. */
+    DynamicPrediction = ovrHmdCap_DynamicPrediction,
+
+    /** Supports rendering without VSync for debugging. */
+    NoVSync = ovrHmdCap_NoVSync
+};
+
+/**
+@brief Hmd capabilities flags
+*/
+typedef Containers::EnumSet<HmdCapability> HmdCapabilities;
+
+enum class HmdTrackingCapability: UnsignedInt {
+    /** @brief Supports orientation tracking (IMU). */
+    Orientation = ovrTrackingCap_Orientation,
+    /** @brief Supports yaw drift correction via a magnetometer or other means. */
+    MagYawCorrection = ovrTrackingCap_MagYawCorrection,
+    /** @brief Supports positional tracking. */
+    Position = ovrTrackingCap_Position,
+};
+
+/**
+@brief Hmd tracking capabilities flags
+*/
+typedef Containers::EnumSet<HmdTrackingCapability> HmdTrackingCapabilities;
+
+enum class HmdStatusFlag: UnsignedByte {
+    /** A mirror texture was created for the hmd and needs to
+     * be destoryed on destruction of the hmd.  */
+    HasMirrorTexture = 1 << 0,
+    /** The hmd was created as a debug hmd (without real hardware). */
+    Debug = 1 << 1
+};
+
+/**
+@brief Hmd tracking capabilities flags
+*/
+typedef Containers::EnumSet<HmdStatusFlag> HmdStatusFlags;
+
```

Comment:
There should be also the following lines:

```cpp
CORRADE_ENUMSET_OPERATORS(HmdCapabilities)
CORRADE_ENUMSET_OPERATORS(HmdTrackingCapabilities)
CORRADE_ENUMSET_OPERATORS(HmdStatusFlags)
```

Otherwise code like `HmdCapability::LowPersistence|HmdCapability::NoVSync` wouldn't compile.